### PR TITLE
Updates regarding DAA v4

### PR DESCRIPTION
**Pull request by mtreadwa**
-Added the pdf of DAA version 4 to the files folder.
-Made changes to page number references, ordering of steps, and the download link on the How to Fill Out DAA page.
-updated download link of FAQ page
-added a download link on Glossary page

### CODE REVIEW

**Comment by kirbyjbirch** [comment]
Did you redo the screenshots as well?

**Comment by mtreadwa** [comment]
@kirbyjbirch, oops no I didn't realize there add been any changes to that section. I'll redo that now and make another commit.

**Comment by lnason** [comment]
Add explanation of Real Time Clinical Genomics in the Clinical Genomics paragraph

File: mkdocs.yml [hunk 1]
```diff
@@ -14,6 +14,7 @@ pages:
   - 'Welcome to St. Jude Cloud': 'index.md'
   - 'Create An Account': 'create-an-account.md'
   - 'Data Guides': 
+      - 'Studies': 'guides/data/studies.md'
       - 'Types of Data': 'guides/data/types-of-data.md'
```
Comment (lnason) [inline mkdocs.yml]: Is this the order it shows in? It needs to go under "types of data" and above "metadata"

**Pull request merged**